__tests__/Container.test.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import {shallow} from 'enzyme';
+
+import {decorators} from '../src';
+import Container from '../src/components/Decorators/Container';
+import animations from '../src/themes/animations';
+import data from '../example/data';
+import style from '../src/themes/default';
+
+const onClick = jest.fn();
+
+const renderComponent = (props = {}) => <Container
+    {...{decorators, onClick, animations}}
+    node={data}
+    style={style.tree.node}
+    {...props}
+/>;
+
+describe('<Container/>', () => {
+    it('should return children with contains decorators.Header if terminal prop is true', () => {
+        const wrapper = shallow(renderComponent({terminal: true}));
+        expect(wrapper.children().contains(<decorators.Header node={data} style={style.tree.node.header}/>)).toBe(true);
+    });
+    it('should exists Toggle component if terminal and animations are false', () => {
+        const wrapper = shallow(renderComponent({terminal: false, animations: false}));
+        expect(wrapper.find('Toggle').exists()).toBe(true);
+    });
+    it('should exists VelocityComponent if terminal is false and animations exists animations', () => {
```

Comment:
the `if terminal is false and animations exists animations` should be moved to `describe` sentences.